src/application/metrics.rs

```diff
@@ -1,5 +1,5 @@
 #![allow(dead_code)]
```

Comment:
since metrics are used in code, pls remove this:
`#![allow(dead_code)]
`
also there is no need to put 
`#[inline(always)]
`
it is better to let compiler to decide when to inline. Just pls remove those, this is old code.